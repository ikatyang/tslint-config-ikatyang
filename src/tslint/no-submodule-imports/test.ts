// good

import something from 'a';

// bad

import something from 'a/b';
