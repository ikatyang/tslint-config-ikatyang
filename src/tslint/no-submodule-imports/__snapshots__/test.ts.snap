// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

import something from 'a';

// bad

import something from 'a/b';
                      ~~~~~  [Submodule import paths from this package are disallowed; import from the root instead]

`;
